CondCore/CondDB/test/condTestRegression.py

```diff
@@ -9,13 +9,25 @@
 import re
 import json
 
-readers = { 'CMSSW_9_0_1'        : ['slc6_amd64_gcc530'], 
+# Boost 1.51 [no writers available - a back-port is required!]
+#readers = { 'CMSSW_7_1_0'        : ['slc6_amd64_gcc490'], 
+#          }
+
+# Boost 1.57
+#readers = { 'CMSSW_7_5_0'        : ['slc6_amd64_gcc491'], 
+#          }
+
+# Boost 1.63
+#readers = { 'CMSSW_9_0_0'        : ['slc6_amd64_gcc530'], 
+#          }
+
+# Boost 1.67 [No reference release yet...]
+readers = {
```

Comment:
@ggovi so at present this just picks the current architecture of the used build, am I correct?

Reply:
yes, there is no full release yet available as a reader, there will be one after a full 10_4 will be available.